### DISCUSSION TITLE
How to display repeater field values with acf as an array with GraphQL.

### DISCUSSION THREAD

**Discussion by antizhyk** [wp-graphql/wp-graphql#2817] [Advanced Custom Fields]
Please help me, I created a group of fields with the repeater type. I was hoping to get an array of values. But instead the values of this repeater GraphQL gives them to me as an object while mixing with the rest of the fields.

![2023-05-12_20-37](https://github.com/wp-graphql/wp-graphql/assets/78226834/e3d93546-1e1e-4cf7-8a91-56406de16961)

![2023-05-12_20-41](https://github.com/wp-graphql/wp-graphql/assets/78226834/48de57c0-b0f8-4459-a33c-c10f079eadb8)

**Comment by whatevercauseeverythingistaken** [comment]
I know it's been quite some time but have you managed to fix this?

**Comment by jasonbahl** [comment]
@antizhyk @whatevercauseeverythingistaken have ya'll checked out the new WPGraphQL for ACF? We recently re-architected the plugin and it's now on the WordPress.org repo and has a new home on GitHub as well: https://github.com/wp-graphql/wpgraphql-acf. 

You can read more here: https://acf.wpgraphql.com and if you're upgrading from an older version we have an upgrade guide here: https://acf.wpgraphql.com/upgrade-guide. 

A lot of issues that existed in the old version of the plugin have been resolved. 

If this is still an issue for you, I'd recommend opening an issue on that repo with clear steps to reproduce and include things like versions of plugins you are running, an ACF Field Group export, queries you're trying to run, etc.
